File: app/src/main/java/org/wcl/mfn/app/MFNHelperApplication.java
package org.wcl.mfn.app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;

@SpringBootApplication(scanBasePackages = "org.wcl.mfn")
@EnableWebMvc
public class MFNHelperApplication {

    public static void main(String[] args) {
        SpringApplication.run(MFNHelperApplication.class, args);
    }
}

File: ui/src/main/kotlin/org/wcl/mfn/ui/app/MFNHelperApplication.kt
package org.wcl.mfn.ui.app

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.web.servlet.config.annotation.EnableWebMvc
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["org.wcl.mfn"])
@EnableWebMvc
open class MFNHelperApplication
    fun main(args: Array<String>) {
        runApplication<MFNHelperApplication>(*args)
    }
